src/ce/crt0.src

```diff
@@ -438,19 +488,20 @@ end virtual
 _abort:
 	ld	hl,1 shl 23+6	; SIGABRT
 
-	section	.start
+	section	.fini
 	private	skip._abort
 skip._abort:
+	pop	hl
```

Comment:
This doesn't make any sense once you fix the section bugs.
```suggestion
```

Reply:
fixed and updated